components/user-profile-ui/src/main/resources/XWiki/AdminUserProfileSheet.xml

```diff
@@ -47,7 +47,7 @@
 #set ($sectionsObjectClassName = 'XWiki.UserProfileSectionsClass')
 #set ($sectionObjectClassName = 'XWiki.UserProfileSectionClass')
 ##
-#set ($xredirect = $doc.getURL($xcontext.action, ${request.queryString}))
+#set ($xredirect = $escapetool.xml($escapetool.url($doc.getURL($xcontext.action, ${request.queryString}))))
```

Comment:
The XML escape isn't needed, since all URL characters are safe; `<&>` are already escaped by the inner call.